Source/SwiftLintFramework/Rules/Style/DisallowTopLevelVariablesRule.swift

```diff
@@ -0,0 +1,64 @@
+import SwiftSyntax
+
+struct DisallowTopLevelVariablesRule: SwiftSyntaxRule, OptInRule, ConfigurationProviderRule {
+    var configuration: SeverityConfiguration = .init(.warning)
```

Comment:
A matter of taste, but the preferred/consistent style would be `var configuration = SeverityConfiguration(.warning)`.